bench/pbrt-v4/optimized/shapes.cpp.ll

```diff
@@ -7210,21 +7210,22 @@ if.then133:                                       ; preds = %for.end
   %div = fdiv float %mul142, %mul143
   %call144 = call noundef i32 @_ZN4pbrt7Log2IntEf(float noundef %div)
   %div145 = sdiv i32 %call144, 2
-  %high.val.i = call i32 @llvm.smin.i32(i32 %div145, i32 10)
-  %retval.0.i = call i32 @llvm.smax.i32(i32 %high.val.i, i32 0)
+  %cmp.i144 = icmp slt i32 %call144, -1
```

Comment:
It breaks smax SPF. Should be fixed by https://github.com/llvm/llvm-project/pull/82472.